README.md

```diff
@@ -42,157 +42,161 @@ Optionally include the following parameters when running the cli:
 ```
 
 ### get-seat-info
 Returns the status and address of a trader's seat. By default, returns the payer's seat info. Seats can be in the following states: Approved, NotApproved, Retired
 
-`$ phoenix-cli -u dev get-seat-info -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8 -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
+`$ phoenix-cli -u main get-seat-info -m 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg -t 3HBWHuyxWv4uN8U8SeukocrWPfLZJqrtj9DgDHsGo2HR`
 ```
-Seat address: mg6uXraBkvi7ccbnKvJoXgyjUVDBSqcxDJysPqiMYau
+Seat address: GGyZqgoqnKsvMTsmSSkTrDjtdSFUsEoioKz9Yr2vEnZa
 Seat status: Approved
 ```
 
 ### get-open-orders
 Returns all open orders on a given market for a trader. By default, returns the payer's open orders. Returns the side, orderID, price in ticks, price, and size for each order. 
 
-`$ phoenix-cli -u dev get-open-orders -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8 -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
+`$ phoenix-cli -u main get-open-orders 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
 ```
 Open Bids
 ID                   | Price (ticks)        | Price      | Quantity  
-18446744073708399662 | 2385                 | 11.925     | 5.870     
-18446744073708399660 | 2384                 | 11.920     | 15.100    
-18446744073708399658 | 2382                 | 11.910     | 30.226    
-18446744073708399656 | 2380                 | 11.900     | 67.226    
-18446744073708399654 | 2377                 | 11.885     | 111.064   
+18446744073708399662 | 2385                 | 22.925     | 5.870     
+18446744073708399660 | 2384                 | 22.920     | 15.100    
+18446744073708399658 | 2382                 | 22.910     | 30.226    
+18446744073708399656 | 2380                 | 22.900     | 67.226    
+18446744073708399654 | 2377                 | 22.885     | 111.064   
 
 Open Asks
 ID                   | Price (ticks)        | Price      | Quantity  
-1151952              | 2387                 | 11.935     | 4.865     
-1151954              | 2388                 | 11.940     | 15.075    
-1151956              | 2390                 | 11.950     | 30.125    
-1151958              | 2392                 | 11.960     | 66.889    
-1151960              | 2395                 | 11.975     | 110.229   
-1151962              | 2399                 | 11.995     | 166.736   
+1151952              | 2387                 | 22.935     | 4.865     
+1151954              | 2388                 | 22.940     | 15.075    
+1151956              | 2390                 | 22.950     | 30.125    
+1151958              | 2392                 | 22.960     | 66.889    
+1151960              | 2395                 | 22.975     | 110.229   
+1151962              | 2399                 | 22.995     | 166.736   
 ```
 
 ### request-seat
 Requests a seat for the payer on the given market. Note that the seat will have to then be approved by the market authority in order to place limit orders. 
```

Comment:
add a note here that this will send a tx to the network, and also charge you X rent